src/ConnectToLedger.ts

```diff
@@ -32,13 +34,19 @@ const qldbDriver: QldbDriver = createQldbDriver();
  */
 export function createQldbDriver(
     ledgerName: string = LEDGER_NAME,
-    serviceConfigurationOptions: ClientConfiguration = {}
+    serviceConfigurationOptions: QLDBSessionClientConfig = {region: AWS_REGION}
```

Comment:
Nitpick - I see elsewhere in this repo we have added spaces between curly brackets and object key/value pairs. For consistency we might want to do the same here (and elsewhere in this PR), so this would become `serviceConfigurationOptions: QLDBSessionClientConfig = { region: AWS_REGION }`.

It might be a good idea to add this to our eslint config if you don't mind: https://eslint.org/docs/latest/rules/object-curly-spacing

Reply:
Yeah, I agree